src/xpcc/architecture/driver/accessor/flash_reader.hpp

```diff
@@ -75,6 +75,7 @@
 		#define	EXTERN_FLASH_STORAGE_STRING(s)	extern const char s[]
 
 		#define	INLINE_FLASH_STORAGE_STRING(s)	((const char *)(s))
```

Comment:
I don't know what I was smoking when I thought that `INLINE_FLASH_STORAGE_STRING("Hello World!")` was an acceptable length for a macro to be used inline.